src/main/java/uk/gov/beis/els/categories/spaceheaters/model/TankLabelClass.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.beis.els.categories.spaceheaters.model;
+
+import java.util.Arrays;
+import uk.gov.beis.els.model.Displayable;
+
+public enum TankLabelClass implements Displayable {
+  AP("A+", "A", "+", 0.95F),
+  A("A", "A", "", 0.91F),
+  B("B", "B", "", 0.86F),
+  C("C", "C", "", 0.83F),
+  D("D", "D", "", 0.81F),
+  E("E", "E", "", 0.81F),
+  F("F", "F", "", 0.81F),
+  G("G", "G", "", 0.81F);
+
+  private final String displayName;
+  private final String letter;
+  private final String plusses;
+  private final float ratingValue;
+
+  TankLabelClass(String displayName, String letter, String plusses, float ratingValue) {
+    this.displayName = displayName;
+    this.letter = letter;
+    this.plusses = plusses;
+    this.ratingValue = ratingValue;
+  }
+
+  @Override
+  public String getDisplayName() {
+    return displayName;
+  }
+
+  public String getLetter() {
+    return letter;
+  }
+
+  public String getPlusses() {
+    return plusses;
+  }
+
+  public float getRatingValue() {
+    return ratingValue;
+  }
+
+  public static TankLabelClass getEnum(String tankLabelClass) {
+    return Arrays.stream(TankLabelClass.values())
+        .filter(e -> e.getDisplayName().equals(tankLabelClass) || e.name().equals(tankLabelClass))
+        .findFirst()
+        .orElseThrow(() -> new RuntimeException(String.format("Can't resolve RatingClass from string %s", tankLabelClass)));
```

Comment:
Message should be for the `TankLabelClass`

Reply:
done